### DISCUSSION TITLE
Editing or adding cheats , apollo save tool ps3 hen

### DISCUSSION THREAD

**Discussion by Simon-ghoster** [bucanero/apollo-ps3#156] [General]
Hi, i have a question : is there a way to edit a code that's already available for the game in save patches or add my own new cheat code in apollo save tool ps3 ?! 

Regards.

**Comment by bucanero** [comment]
you can add or edit cheat codes by editing the `.savepatch` text files for the game's TITLE-ID.
Check the [wiki link](https://github.com/bucanero/apollo-ps3/wiki) for details about the text file format.

As reference, PS3 patch files can be found here: https://github.com/bucanero/apollo-patches/tree/main/PS3

**Reply by Simon-ghoster** [reply]
Thanks, I'll check.
